app/src/main/java/com/example/android/picover/presentation/MainActivity.kt

```diff
@@ -0,0 +1,77 @@
+package com.example.android.picover.presentation
```

Comment:
There is still a problem with `MainActivity` because it is shown as deleted and added a new file :D. As I said in another comment – probably the change of package should be done in another commit (and another pull request).

Reply:
I think now it's done. Also, I changed PR to have only one commit and I will create a new PR with moving `MainActivity`.